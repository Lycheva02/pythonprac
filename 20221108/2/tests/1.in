eto ne cifry no zato bukvy
(0,0),(0,2),(2,0)

