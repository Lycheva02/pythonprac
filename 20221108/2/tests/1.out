Invalid input
2.00
